app/grandchallenge/algorithms/admin.py

```diff
@@ -54,6 +60,7 @@ class AlgorithmAdmin(GuardedModelAdmin):
         "title",
         "created",
         "public",
+        "default_io",
```

Comment:
Aren't the inputs and outputs still editable in the algorithm admin?

Reply:
True. I made them read only now.